Makefile

```diff
@@ -381,15 +381,18 @@ watcher: export CATALOG_IMG=${CATALOG_IMAGE}
 watcher: ## Install watcher operator via olm
 	bash ci/olm.sh
 	oc apply -f ci/olm.yaml
+	timeout 300s bash -c "while ! (oc get csv -n  openshift-operators -l  operators.coreos.com/cluster-observability-operator.openshift-operators -o jsonpath='{.items[*].status.phase}' | grep Succeeded); do sleep 10; done"
```

Comment:
this is ok for now,
i think we can use oc wait to do this as well but this is what we are already doing below so its following the existing convention.

i don't know if that will work with the csv but for the openstackcontrolplane CR we normally do something like this

```
oc wait -n $(KUTTL_NAMESPACE) openstackcontrolplane openstack --for condition=Ready --timeout=300s
```

you can see how that works by looking at the kuttl targets below.

Reply:
I tried with `oc wait` first but it failed because the csv itself takes some time to be created so it does not exist when the command is executed and failed.

The alternative would be to first wait until the csv is created and then use the oc wait you mentioned. I found this way eaiser and consistent with the existing check.

Reply:
ack